src/utils/classifier_report.py

```diff
@@ -0,0 +1,108 @@
+from airium import Airium
+from pathlib import Path
+from csv import DictReader
+import webbrowser
+import argparse
+import os
+import re
+
+
+class ReportGenerator:
+
+    # parses csv and creates a list of dictionaries  
+    def parse_csv(self, csv_path):
+        
+        with open(csv_path, 'r', encoding='utf-8-sig') as f:
+            self.data = list(DictReader(f))
+            
+    def normalize_urls(self, http_url):
+        for item in self.data:
+            item["normalized_path"] = re.sub(r'.+/(?=.+)', http_url, item["normalized_path"])
```

Comment:
I don't think this quite works how we want it to. For a file path like:
`/opt/data/color_bars/shared/normalized/mnt/locos/rbc/11500_popayan/11500_0006/11500_0006_0001.jpg`
we want to produce the URL:
`https://dcr-test.lib.unc.edu/shared/normalized/mnt/locos/rbc/11500_popayan/11500_0006/11500_0006_0001.jpg`
whereas the current regular expression produces:
`https://dcr-test.lib.unc.edu/shared/11500_0006_0001.jpg`

So we need to replace `/opt/data/color_bars/` with http_url, and need to retain the intermediate path information.

Reply:
@bbpennel Just to make sure, '/shared/' should be used as the indicator to replace the parent paths?

Just want to verify, so  I can change the normalized_paths in  `report_test.csv` since the regex won't replace any part of the `ml-repo-preingest-processing/fixtures/normalized_images/ncc/<image file> `

Reply:
That holds true in this case. What would you think of adding another commandline option for specifying the substring that terminates the area that should be replaced by the normalized-url? It could default to `/shared/`.

Reply:
Good idea. I'll add that in.